### PR TITLE
Add xdebug environment variables

### PR DESCRIPTION
**Pull request by duboiss**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -1,6 +1,6 @@
 DOCKER_PHP_IMAGE=8.1
 DOCKER_MYSQL_IMAGE=8.0
-DOCKER_ELASTICSEARCH_IMAGE=7.17
+DOCKER_ELASTICSEARCH_IMAGE=7.6
```
Comment (Fulmenef) [inline .env.dist:3]: You probably want to update this line

Comment (duboiss) [reply .env.dist:3]: ?

Comment (Fulmenef) [reply .env.dist:3]: You changed it last week https://github.com/EmakinaFR/docker-magento2/pull/85/files to keep up-to-date with Magento, 7.6 is not supported anymore in the recent versions of Magento

Comment (duboiss) [reply .env.dist:3]: Oh yes. I didn't understand why you said "want" instead of "don't want" in your previous comment :)

**Review by Fulmenef — APPROVED** [review]

**Pull request merged**